ast/ast.go

```diff
@@ -2390,13 +2391,14 @@ type DropProtoBundle struct {
 //	{{if .PrimaryKeys}}PRIMARY KEY ({{.PrimaryKeys | sqlJoin ","}}){{end}}
 //	{{.Cluster | sqlOpt}}
 //	{{.CreateRowDeletionPolicy | sqlOpt}}
+//	{{.Options | sqlOpt}}
```

Comment:
I'm not sure, but it seems incorrect to me. Is the following suggestion right?

```suggestion
//	{{if .Options}}, {{.Options | sqlOpt}}{{end}}
```

Reply:
You are right. Thank you!